bindings/experimental/distrdf/python/DistRDF/__init__.py

```diff
@@ -137,3 +142,184 @@ def create_distributed_module(parentmodule):
     distributed.VariationsFor = VariationsFor
 
     return distributed
+
+
+@singledispatch   
+def LiveVisualize(drawable_callback_dict: Dict[type, Optional[Callable]], global_callback: Optional[Callable] = None):
```

Comment:
I think all the `LiveVisualize`-related functions should live in their own separate module. Then in the `__init__.py` you just import the `LiveVisualize function and make it available for users.